homeassistant/components/keymitt_ble/translations/en.json

```diff
@@ -0,0 +1,37 @@
+{
+  "config": {
```

Comment:
Don't forget to regenerate this

Reply:
Sorry what am I regenerating?

Reply:
`python3 -m script.translations develop`